### PR TITLE
Fix issue where save to db wouldn't work in ECR viewer

### LINKED ISSUE #1271: Bug fix: writing to ecr-viewer database from orchestration
## Action Requested
Ensure eCRs passed to the orchestration service can be rendered in the ecr-viewer.

## Acceptance Criteria
Upon upload of an eCR ZIP file to the `/process` endpoint, a database record is written.

## Testing
This functionality should be added to integration tests for the orchestration service.

## Background & Context
There seems to be a bug in the service, such that UUIDs can't be extracted from the eCR and used as the database key.

### PR DESCRIPTION
**Pull request by nickbristow**
# PULL REQUEST

## Summary
There was an issue where save to db would just save the response of the previous service causing issues with the ECR_VIEWER. This change makes it so the bundle is saved instead of the response

## Related Issue
Fixes #1271

## Additional Information
Anything else the review team should know?

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/orchestration/app/services.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from fastapi import Response
 from fastapi import WebSocket
 from fastapi.encoders import jsonable_encoder
+from icecream import ic
```
Comment (emmastephenson) [inline containers/orchestration/app/services.py:12]: We can probably remove the icecream logs now

**Review by emmastephenson — COMMENTED** [review]
Changes largely LGTM, but we should also run this by the VIPER team - @bamader in particular has been working on recent changes to the orchestration service to make it more iteroperable with different configurations, so would be good to get his eyes on it.

Brandon for your SA this is a temporary change - we're still working to move the `save_to_db` functionality to the eCR Viewer, but in the interim the bundle wasn't saving correctly.

File: containers/orchestration/app/services.py (hunk 1 above)
Comment (nickbristow) [reply containers/orchestration/app/services.py:12]: i'll miss it

**Comment by bamader** [comment]
I _think_ the ideas in this PR won't mess up anything I'm doing, but I did just open a PR to completely overwrite call_apis to use new and much better service handlers, so we're going to run into merge conflicts between these two. The tests and data themselves weren't actually doing anything at all so those changes shouldn't matter to much, but yeah I think this is just going to cause some conflicts in call apis. Nothing too hard to sift out though.

File: containers/orchestration/app/default_configs/sample-orchestration-config-new.json
Comment (emmastephenson) [inline containers/orchestration/app/default_configs/sample-orchestration-config-new.json:1]: Is deleting this file intentional? I thought this was the one Brandon and the rest of VIPER are using.

File: containers/orchestration/app/services.py (hunk 1 above)
Comment (emmastephenson) [reply containers/orchestration/app/services.py:28]: Please? :) Looks like it's still in the file

File: containers/orchestration/app/utils.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 from fastapi import UploadFile
 from fastapi.encoders import jsonable_encoder
 from fastapi.responses import JSONResponse
+from icecream import ic
```
Comment (emmastephenson) [inline containers/orchestration/app/utils.py:15]: Another ic to remove, please

File: containers/orchestration/app/default_configs/sample-orchestration-config-new.json
Comment (emmastephenson) [reply containers/orchestration/app/default_configs/sample-orchestration-config-new.json:1]: Looked through the rest of the PR, saw that this has been replaced with the old file name - is the goal to just go back to a single unified config?

Comment (lina-roth) [reply containers/orchestration/app/default_configs/sample-orchestration-config-new.json:1]: I believe that we noticed that the old configuration wasn't making it through and when we used the new configuration it worked. And while structurally different, the content was the same, so I think we just decided to use that configuration everywhere. @nickbristow care to comment?

File: containers/orchestration/app/utils.py (hunk 2 above)
Comment (lina-roth) [reply containers/orchestration/app/utils.py]: Fixed!

**Review by emmastephenson — APPROVED** [review]

**Pull request merged**